sentinelhub/aws_safe.py

```diff
@@ -71,25 +103,11 @@ def get_safe_struct(self):
                     metafiles=self.metafiles,
                     data_collection=self.data_collection,
                 ).get_safe_struct()
-                for tile_name, safe_struct in tile_struct.items():
-                    safe[main_folder][AwsConstants.GRANULE][tile_name] = safe_struct
-
-        safe[main_folder][AwsConstants.HTML] = {}  # AWS doesn't have this data
-        safe[main_folder][AwsConstants.INFO] = {}  # AWS doesn't have this data
-
-        safe[main_folder][self.get_product_metadata_name()] = self.get_url(AwsConstants.METADATA)
-        safe[main_folder]["INSPIRE.xml"] = self.get_url(AwsConstants.INSPIRE)
-        safe[main_folder][self.add_file_extension(AwsConstants.MANIFEST)] = self.get_url(AwsConstants.MANIFEST)
 
-        if self.is_early_compact_l2a():
-            safe[main_folder]["L2A_Manifest.xml"] = self.get_url(AwsConstants.L2A_MANIFEST)
-            safe[main_folder][self.get_report_name()] = self.get_url(AwsConstants.REPORT)
+                for tile_name, safe_struct in tile_struct.items():
+                    granule_safe[tile_name] = safe_struct
```

Comment:
i think this can be `granule_safe.update(tile_struct)`

Reply:
nice catch